lib/price-import.coffee

```diff
@@ -83,8 +83,8 @@ class PriceImport extends ProductImport
 
   _resolvePriceReferences: (price) =>
     Promise.all [
-      @_resolveReference(@client.customerGroups, 'customerGroup', price.customerGroup, "name=\"#{price.customerGroup?.id}\"")
-      @_resolveReference(@client.channels, 'channel', price.channel, "key=\"#{price.channel?.id}\"")
+      @_resolveReference(@client.customerGroups, 'customerGroup', price.customerGroup, "name=\"#{price.customerGroup?.groupName}\"")
+      @_resolveReference(@client.channels, 'channel', price.channel, "key=\"#{price.channel?.key}\"")
```

Comment:
How is your third point different from what i suggest to do now? 
Also approach with IDs has been done on purpose so that reference syntax for resolution is always the same (https://github.com/sphereio/sphere-product-import/search?utf8=%E2%9C%93&q=_resolveReference) and more user friendly (internally ID might be uuid, key, whatever - could be configurable) and at some point we planned to make schemas for our importers public: https://github.com/sphereio/sphere-json-schemas/blob/master/schema/price.schema.json